Client/Frontend/Browser/BrowserViewController/BrowserViewController+URLBarDelegate.swift

```diff
@@ -135,21 +135,38 @@ extension BrowserViewController: URLBarDelegate {
             return
         }
 
-        let contextualViewProvider = ContextualHintViewProvider(forHintType: .shoppingExperience,
-                                                                with: profile)
-
-        let contextHintVC = ContextualHintViewController(with: contextualViewProvider)
+        if shoppingContextHintVC == nil {
+            configureShoppingContextVC(at: sourceView)
```

Comment:
Just trying to understand; we need a new `ContextualHintViewController` for each CFR we show, aka a `ContextualHintViewController` cannot be reused to show two times the same CFR type?